xml/System.Threading/ThreadPool.xml

```diff
@@ -1285,11 +1285,11 @@ If a thread pool implementation may have different types of threads, the count i
         <Parameter Name="preferLocal" Type="System.Boolean" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="callBack">To be added.</param>
-        <param name="preferLocal">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="callBack">A <see cref="T:System.Threading.ThreadPool.IThreadPoolWorkItem" /> that represents the work item to invoke when a thread in the thread pool picks up the work item.</param>
+        <param name="preferLocal"><see langword="true" /> to prefer queueing the work item in a queue close to the current thread; <see langword="false" /> to prefer queueing the work item to the thread pool's shared queue.</param>
+        <summary>Queues the specified work item object to the thread pool.</summary>
+        <returns><see langword="true" /> if the method succeeds; <see cref="T:System.OutOfMemoryException" /> is thrown if the work item could not be queued.</returns>
+        <remarks>The thread pool will invoke the work item's <see cref="T:System.Threading.ThreadPool.IThreadPoolWorkItem.Execute" /> method.  It is the responsibility of that work item to propagate <see cref="T:System.Threading.ExecutionContext" /> if it's needed; the thread pool will not do so.</remarks>
```

Comment:
Thanks for the change, Stephen. I updated this remark to the correct formatting:

```suggestion
        <remarks>
          <format type="text/markdown"><![CDATA[

## Remarks

The thread pool will invoke the work item's <xref:System.Threading.ThreadPool.IThreadPoolWorkItem.Execute> method.  It is the responsibility of that work item to propagate <xref:System.Threading.ExecutionContext> if it's needed; the thread pool will not do so.

         ]]></format>
       </remarks>
```